Modules/Filtering/FastMarching/include/itkFastMarchingUpwindGradientImageFilter.h

```diff
@@ -240,7 +237,7 @@ class ITK_TEMPLATE_EXPORT FastMarchingUpwindGradientImageFilter : public FastMar
    *  Returns true if at least a target point exists; returns false otherwise.
    */
   bool
-  IsTargetPointsExistenceConditionSatisfied()
+  IsTargetPointsExistenceConditionSatisfied() const
```

Comment:
To me, this is not related to the bug itself, and I'd advocate for adding it in a separate commit.

Reply:
VerifyPreconditions is a const member function and it calls this method, so it needed to be changed.